packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -134,7 +149,9 @@ export function createOdspNetworkError(
 
     error.online = OnlineStatus[isOnline()];
 
-    const props: ITelemetryProperties = { response: responseText };
+    const facetCodes = responseText !== undefined ? parseFacetCodes(responseText) : undefined;
+    const props: ITelemetryProperties = { response: responseText,
+        facetCode: facetCodes !== undefined ? facetCodes[0] : undefined};
```

Comment:
You set only facetCode property, but expect facetCodes (plural) in connectToDeltaStream()

Reply:
the ITelemetryProperties does not have a type string[], I can just add it?

Reply:
can i just do this `error.facetCodes = facetCodes;`

Reply:
@markfields - what do you think?
I think for now it's fine to go with property like you suggested (I believe it should work, i.e. show up in telemetry in some form), but long term I see two approaches:
- we add array support for logging object / telemetry (i.e. make it formal, after validating it works).
- we finally separate logging properties vs. object properties. That way we would use string (stringified payload) for telemetry, and property on object (array) for internal consumption. Today, it's one and the same, unfortunately (you can blame me for that :))

We can follow up later on long-term support (outside of this PR)